designer/src/main/resources/validation/rvdproject/1.11/gatherstep.json

```diff
@@ -31,16 +32,29 @@
 					"items": {
 						"type": "object",
 						"properties": {
-							"digits": {"type": "string","minLength":1},
+							"key": {"type": "string","minLength":1},
 							"next": {"type": "string","minLength":1},
 							"scope": {"type": "string"}
 						},
-						"required":["digits","next"]
+						"required":["key","next"]
 					},
-					"minItems": 1
+					"minItems": 0
+				},
+				"speechMapping": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"properties": {
+							"key": {"type": "string","minLength":1},
+							"next": {"type": "string","minLength":1},
+							"scope": {"type": "string"}
+						},
+						"required":["key","next"]
+					},
+					"minItems": 0
 				}
 			},
-			"required":["mappings"]
+			"required":["mappings", "speechMapping"]
```

Comment:
Hmm, maybe remove all `required` *mapping fields here and just issue warning on client side.